src/containers/Application/Part1.js

```diff
@@ -1,6 +1,424 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { H1, QuestionHeading, CenteredH1 } from '../../components/Typography'
+import { TextInput } from '../../components/Input/TextInput'
+import Dropdown from '../../components/Input/Dropdown'
+import Select from '../../components/Input/Select'
+import { FormSpacing } from '../../components/ApplicationForm'
+
+const genderOptions = [
+  { value: 'female', label: 'Female' },
+  { value: 'male', label: 'Male' },
+  { value: 'non-binary', label: 'Non-binary' },
+  { value: 'other', label: 'Other' },
+  { value: 'prefer not to say', label: 'Prefer not to say' },
+]
+
+const educationOptions = [
+  { value: 'high school', label: 'High school' },
+  { value: 'undergraduate', label: 'Undergraduate' },
+  { value: 'graduate', label: 'Graduate or Post-Graduate' },
+  { value: 'other', label: 'Other' },
+]
+
+// check w logs
+const graduationOptions = [
+  { value: 2019, label: '2019' },
+  { value: 2020, label: '2020' },
+  { value: 2021, label: '2021' },
+  { value: 2022, label: '2022' },
+  { value: 2023, label: '2023' },
+  { value: 2024, label: '2024' },
+  { value: 2025, label: '2025+' },
+]
+
+// check w logs
```

Comment:
might be good to just do 0, 1, 2, 3, 4+ cause anything past that is pretty much the same :angrycry:

Reply:
roger that!